src/mds/Server.cc

```diff
@@ -578,6 +577,39 @@ void Server::handle_client_session(const cref_t<MClientSession> &m)
   if (logger)
     logger->inc(l_mdss_handle_client_session);
 
+  auto& addr = session->info.inst.addr;
```

Comment:
Why move this block is this is not being carved out as a separate helper (for other potential callers)?

Reply:
This was in case CEPH_SESSION_REQUEST_OPEN and I want to use it default case too. Thats the only reason.

Reply:
OK. Thx.